fmn/consumer/worker.py

```diff
@@ -1,24 +1,23 @@
 # FMN worker figuring out for a fedmsg message the list of recipient and
 # contexts
 
-
 import json
 import logging
 import time
 import random
 
+from dogpile.cache import make_region
+from fedmsg_meta_fedora_infrastructure import fasshim
```

Comment:
That's definitely the general ordering.

The sub-ordering I chose here is purely based on ordering alphabetically, but I have no preference.